util/pkg/vfs/s3fs.go

```diff
@@ -155,6 +163,50 @@ func (p *S3Path) WriteFile(data io.ReadSeeker, aclObj ACL) error {
 	return nil
 }
 
+func (p *S3Path) GetVersioning() (bool, error) {
+	client, err := p.client()
+	if err != nil {
+		return false, err
+	}
+
+	request := &s3.GetBucketVersioningInput{}
+	request.SetBucket(p.bucket)
+
+	response, err := client.GetBucketVersioning(request)
+	if err != nil {
+		return false, fmt.Errorf("error getting s3 bucket versioning: %v", p, err)
+	}
+
+	if strings.Contains(response.String(),"Status: \"Enabled\"") {
```

Comment:
Is there a value in the AWS APU for this?  I would rather not test a string response.

Reply:
This was the only way I could find. Return value doesn't have a status flag.